lib/marked-renderer.js

```diff
@@ -1,4 +1,15 @@
 const { marked } = require('marked')
+const { gfmHeadingId } = require('marked-gfm-heading-id')
```

Comment:
Unlike the [equivalent issue](https://github.com/alphagov/govuk-frontend/pull/3683) on `govuk-frontend`, I'm installing the plugin here. I think it's much more likely that headings will get included here.